test/e2e/subscription_e2e_test.go

```diff
@@ -2889,9 +2890,9 @@ func fetchSubscription(crc versioned.Interface, namespace, name string, checker
 			return false, err
 		}
 		thisState, thisCSV, thisInstallPlanRef := fetchedSubscription.Status.State, fetchedSubscription.Status.CurrentCSV, fetchedSubscription.Status.InstallPlanRef
-		if thisState != lastState || thisCSV != lastCSV || equality.Semantic.DeepEqual(thisInstallPlanRef, lastInstallPlanRef) {
+		if thisState != lastState || thisCSV != lastCSV || !equality.Semantic.DeepEqual(thisInstallPlanRef, lastInstallPlanRef) {
```

Comment:
Seems fairly important 😉

Reply:
I wrote this bad code last week so thankfully a small blast radius :)